x/group/internal/orm/testsupport.go

```diff
@@ -86,7 +86,8 @@ func NewGasCountingMockContext() *GasCountingMockContext {
 }
 
 func (g GasCountingMockContext) KVStore(store sdk.KVStore) sdk.KVStore {
-	return gaskv.NewStore(store, g.GasMeter, storetypes.KVGasConfig())
+	gasConfig := storetypes.KVGasConfig()
+	return gaskv.NewStore(store, g.GasMeter, gasConfig)
```

Comment:
Why does this offer any performance improvement?

Reply:
the perf improvement is this: https://github.com/cosmos/cosmos-sdk/pull/14203/files#diff-abad3e7f81deb0a089f7e974e60a081e67a7b21e5c6ba5c7b2be3785a66445afR15

this allows us to save time because of alloc + copy, gasKV config is static for a specific context, meaning that as the context is running the gas configuration (how much we spend on each Read/Write/Iter/etc) does not change. 

My benches showed it saved a further 10ns

Reply:
> this allows us to save time because of alloc + copy

Cool, where can we see the benchmark results that demonstrate the perf increases? It would be super cool if changing the representation of this ~50 byte struct from a value to a pointer managed to produce performance benefits from the reduced copy costs that outweighed the additional costs it created on the allocator and GC.